src/tags/views.py

```diff
@@ -58,7 +58,7 @@ def get_context_data(self, **kwargs):
         return super().get_context_data(**kwargs)
```

Comment:
`TagDetailMixin`  isn't a Mixin, it's a new base class that extends the Detail View. Should just be called TagDetailView